samples/snippets/publisher.py

```diff
@@ -212,65 +212,6 @@ def callback(future):
     # [END pubsub_publisher_batch_settings]
 
 
-def publish_messages_with_retry_settings(project_id, topic_id):
-    """Publishes messages with custom retry settings."""
-    # [START pubsub_publisher_retry_settings]
```

Comment:
If this is still needed, we must add support for custom retries to the `client.publish()` method and override the default publish retry settings. Ordering keys will still override the deadline to "infinity", though, which should be documented.

Reply:
Not relevant anymore, added support for custom retries.